### ISSUE TITLE
Interactive window should be more aligned with the accessibility experience of VS Code 

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#161925]
Testing #161850

Since the interactive window is using a lot of custom UI this means that our accessibility users are maybe not used to the interaction it offers.
I tested the improved keyboard navigation in the test plan and it works as expected, but I think we need to take a step back and look at the bigger picture of the interactive window.

For example:
* Can we learn from the Debug Console and Terminal and have similar keyboard interaction as they do since this is something that our users are already familiar with
* Can we use the same navigation as we do in other VS Code toolbars? It is only one tab stop, and left/right are used to navigate through actions
* How do we move focus to the top level toolbar? Does that top level toolbar behave like other toolbars in vscode? Does it have the required aria roles?
* etc...

**Comment by r3m0t** [comment]
Hi, I'm trying to scroll the last output on my interactive (it has a scrollbar). Nothing works, I started with "Interactive Window: Focus History", then I did Ctrl+DownArrow (notebook.cell.focusInOutput), this doesn't seem to do anything in Interactive window.

If I click in the output, PageDown and PageUp are mapped to `list.focusPageDown` and `list.focusPageUp` which scrolls the interactive window instead of the specific output.

**Comment by amunger** [comment]
We've done some work on this for the native REPL through https://github.com/microsoft/vscode/issues/209140, and we'll keep tracking feedback through new issues. Jupyter's interactive window should be migrated to the notebook backed REPL editor to get the full set of a11y functionality
